integration-tests/cypress/e2e/project/deck/deck.cy.ts

```diff
@@ -0,0 +1,244 @@
+
+import { Project } from '../../../pages/Project';
+import { Projects } from '../../../pages/Projects';
+import { isCreateProjectFromTemplateSuccessPayload } from '../../../support/server/createProjectFromTemplateCommand';
+import { Explorer } from '../../../workbench/Explorer';
+
+describe('Verify the Deck Representation', () => {
+  let studioProjectId: string = '';
+  let taskProjectId: string = '';
+  before(() => {
+    // We create the Deck View from the stereotype before executing the tests
+    new Projects().visit();
+    cy.createProjectFromTemplate('blank-studio-template').then((res) => {
+      const payload = res.body.data.createProjectFromTemplate;
+      if (isCreateProjectFromTemplateSuccessPayload(payload)) {
+        studioProjectId = payload.project.id;
+
+        new Project().visit(studioProjectId);
+        cy.getByTestId('new-model').click();
+        cy.getByTestId('name-input').type('Deck View');
+        cy.getByTestId('stereotype').click();
+        cy.get('li').filter(':contains("Deck View")').click();
+        cy.getByTestId('create-document').click();
+      }
+    });
+  });
+  after(() => {
+    // We delete the created studio once all tests have been executed
+    cy.deleteProject(studioProjectId);
+  });
+
+  beforeEach(() => {
+    // We create the Task project from the template before each tests.
+    new Projects().visit();
+    cy.createProjectFromTemplate('task-template').then((res) => {
+      const payload = res.body.data.createProjectFromTemplate;
+      if (isCreateProjectFromTemplateSuccessPayload(payload)) {
+        taskProjectId = payload.project.id;
+        new Project().visit(taskProjectId);
+      }
+    });
+  });
+
+  afterEach(() => {
+    //We delete the task project after each test to make sure that each test do not depend on the previous one.
+    cy.deleteProject(taskProjectId);
+  });
+
+  context('We first verify the Deck Creation, Deletion and renaming', () => {
+    it('We create the Deck Representation', () => {
+      // We create a new empty deck representation
+      const explorer = createDailyDeckRepresentation('Project Dev', 'New Daily Representation');
+      explorer.getSelectedTreeItems().contains('New Daily Representation').should('exist');
+      cy.getByTestId('deck-representation').should('exist');
+    });
+
+    it('We create the Deck Representation on the Daily project', () => {
+      // We create a new deck representation with lanes and cards
+      const explorer: Explorer = createDailyDeckRepresentation('Daily Project Dev', 'New Daily Representation');
+      explorer.getSelectedTreeItems().contains('New Daily Representation').should('exist');
+
+      //We verify that all lanes are present.
+      const days: string[] = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+      days.forEach((day) => {
+        cy.getByTestId('deck-representation').findByTestId(`lane-${day}`).should('exist');
```

Comment:
Same remark with a new function in `Deck.ts`
```typescript
public getLane(laneLabel:string):Cypress.Chainable<JQuery<HTMLElement>> {
  return this.getDeckRepresentation().findByTestId(`lane-${laneLabel}`);
}
```

Reply:
done